firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -307,7 +311,13 @@ private final void doRegistrationInternal(boolean forceRefresh) {
     }
 
     triggerOnStateReached(prefs);
+    // Execute network calls (CreateInstallations or GenerateAuthToken) to the FIS Servers on
+    // a separate executor i.e networkExecutor
+    networkExecutor.execute(() -> fisNetworkCall(forceRefresh));
+  }
 
+  private void fisNetworkCall(boolean forceRefresh) {
```

Comment:
I think fis is redundant here. How about "doNetworkCall".

Reply:
+1.
I already made suggestions in the other thread (see below).
But #doNetWorkCall sounds totally reasonable.

#sendNetworkCall
#makeNetworkCall
#sendRequestToFirebaseInstallationsServerApi
#syncWithFirebaseInstallationsServerApi
#updateLocalAuthToken

Reply:
Done.